doc/command_ref.rst

```diff
@@ -1759,6 +1763,53 @@ of the most recent transaction. The last form is ``last-<offset>``, where
 ``<offset>`` is a positive integer. It specifies offset-th transaction preceding
 the most recent transaction.
 
+.. _excluded_packages-label:
+
+=================
+Package filtering
+=================
+
+Package filtering filters packages out from the available package set, making them invisible to most
+of dnf commands. They cannot be used in a transaction. Packages can be filtered out by either
+Exclude Filtering or Modular Filtering.
+
+-----------------
+Exclude Filtering
+-----------------
+
+Exclude Filtering is a mechanism used by a user or by a DNF plugin to modify the set of available
+packages. Exclude Filtering can be modified by *either :ref:`includepkgs <include-label>` or
+:ref:`excludepkgs <exclude-label>` configuration options in
+:ref:`configuration files <conf_ref-label>`. The -:ref:`-disableexcludes <disableexcludes-label>`
+command line option can be used to override excludes from configuration files. In addition to
+user-configured excludes, plugins can also extend the set of excluded packages. To disable excludes
+from a DNF plugin you can use the -:ref:`-disableplugin <disableplugin-label>` command line option.
+
+To disable all excludes for e.g. the install command you can use the following combination
+of command line options:
+
+``dnf --disableexcludes=all --disableplugin=* install bash``
+
+-----------------
+Modular Filtering
+-----------------
+
+Please see :ref:the modularity documentation <modularity-label> for details on how Modular Filtering
```

Comment:
I think you need the backticks in the `:ref:` here, I've checked the html and it doesn't work like this.